packages/core/utils/src/modules-sdk/abstract-module-service-factory.ts

```diff
@@ -501,7 +503,12 @@ export function abstractModuleServiceFactory<
           await this.eventBusModuleService_?.emit(
             softDeletedEntities.map(({ id }) => ({
               eventName: `${kebabCase(model.name)}.deleted`,
-              data: { id },
+              body: {
+                id,
+                // TODO: fulfil these
```

Comment:
Let's create a ticket for this if we haven't already 🙏

Reply:
this is part of the other events project to find a way to refactor this 👍